source/manual/environments.html.md

```diff
@@ -1,48 +1,83 @@
 ---
-owner_slack: "#govuk-developers"
+owner_slack: "#govuk-platform-engineering"
 title: GOV.UK's environments (integration, staging, production)
 section: Infrastructure
 type: learn
 layout: manual_layout
 parent: "/manual.html"
 ---
 
-GOV.UK has several environments with different purposes.
+GOV.UK has several environments with different purposes. By _environment_, we
+mean:
 
-For a quick view of what's where, you can use the [release app](https://release.publishing.service.gov.uk).
+- a self-contained, running instance of all the software components of GOV.UK
+- the AWS account and VPC network which hosts that instance of GOV.UK
+- the configuration (settings) for that instance of GOV.UK
 
-## Continuous integration (CI)
+This document describes each GOV.UK environment and its uses.
 
-Runs tests for applications.
+There are three GOV.UK environments hosted in AWS: integration, staging and
+production. Individual developers also run GOV.UK locally on laptops or
+workstations.
+
+The [Release dashboard](https://release.publishing.service.gov.uk/) shows which
```

Comment:
I'd move this alias link to the bottom of the doc for consistency.

Reply:
Thanks but I don't see any benefit to that.